config/locales/two_factor_authentication/fr.yml

```diff
@@ -25,7 +24,7 @@ fr:
       instructions: Si vous n’avez pas accès à un autre appareil, conservez vos codes
         de sauvegarde en lieu sûr. Si vous perdez vos codes de sauvegarde, vous
         ne pourrez plus vous connecter à %{app_name}.
-      new_backup_codes_html: J’ai besoin de nouveaux codes de sauvegarde
+      new_backup_codes_html: J’ai besoin de nouveaux codes de sauvegarde&ZeroWidthSpace;
```

Comment:
ok so for this one, the English copy has `&nbsp;` to control word wrapping, but I didn't know the appropriate place to put one in the other languages, so I added `&ZeroWidthSpace;` to the end because it should be invisible and not affect the wrapping

Reply:
it's also the specific string from https://github.com/18F/identity-idp/pull/8708